sympy/printing/lambdarepr.py

```diff
@@ -9,7 +9,8 @@ class LambdaPrinter(StrPrinter):
     This printer converts expressions into strings that can be used by
     lambdify.
     """
-
+    printmethod="_lambda_repr"
```

Comment:
`'_lambdify'` would more closely fit the naming scheme of the other printmethods.